YoCode/FileImport.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace YoCode
+{
+    // To import:
+    // UnitConverter.cs from junior-test/UnitConverterWebApp
+    // Program.cs from junior-test/UnitConverterWebApp
+    // Startup.cs from junior-test/UnitConverterWebApp
+    // HomeController.cs from junior-test/UnitConverterWebApp/Controllers
+
+    class FileImport
+    {
+        public string ORIGINAL_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\junior-test";
+        public string MODIFIED_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\original-test";
```

Comment:
These paths won't work on somebody else's PC. They should either be relative paths so people have a chance to put them in the expected place, or even better they should be parameters that the user can specify.

Reply:
@jogobom I felt that I owe some unit tests, because i've not written any until today